app/views/main.scala.html

```diff
@@ -67,36 +75,23 @@
 
     @inlineScript = {
 
-    @scriptElement
-
-    @if(configDecorator.analyticsToken == Some("N/A")) {
-        <script type="text/javascript">
-                var ga = function (send, event, category, action, label, args) {
-                    console.log(JSON.stringify({
-                        call: 'ga',
-                        send: send,
-                        event: event,
-                        category: category,
-                        action: action,
-                        label: label
-                    }));
-
-                    if (args && typeof args !== 'undefined' && typeof args['hitCallback'] === 'function') {
-                        var hitCallback = args['hitCallback'];
-                        hitCallback();
-                    }
-
-                };
-                window._gaq.push = function (args) {
-                    console.log(JSON.stringify({
-                        call: '_gaq.push',
-                        args: args
-                    }));
-                };
-        </script>
-    }
 
-    @if(displayTimeoutDialog) {
+@scriptElement
+
+@if(configDecorator.analyticsToken == Some("N/A")) {
+    }
```

Comment:
Is this meant to be an empty if? Unless I'm misreading it, this looks like it isn't doing anything. Can you confirm? @brendanstraughan